integreat_cms/static/src/js/pois/poi-map.ts

```diff
@@ -1,165 +1,242 @@
+        document.getElementById("address_loading").classList.add("hidden");
+        // Show found address
+        const street = constructStreet(data.street, data.number);
+        const updatedStreet = document.getElementById("updated_street");
+        if (street) {
+            updatedStreet.textContent = street;
+            updatedStreet.classList.remove("hidden");
         } else {
-            document.getElementById("map").classList.add("hidden");
-            document.getElementById("no_coordinates_alert")?.classList.remove("hidden");
+            updatedStreet.textContent = "";
+            updatedStreet.classList.add("hidden");
         }
-    });
+        document.getElementById("updated_postcode").textContent = data.postcode;
+        document.getElementById("updated_city").textContent = data.city;
+        document.getElementById("updated_country").textContent = data.country;
+        document.getElementById("update_address_from_map_marker").classList.remove("hidden");
+        // Enable address-related radio buttons
+        Array.from(document.getElementsByClassName("depends-on-address")).forEach((element) => {
+            element.classList.remove("cursor-not-allowed", "text-gray-500");
+            if (element instanceof HTMLInputElement) {
+                element.disabled = false;
+            }
+        });
+    } else {
+        // Hide loading spinner
+        document.getElementById("address_loading").classList.add("hidden");
+        // Hide all address-dependent input fields in case they were shown before
+        document.getElementById("update_address_from_map_marker").classList.add("hidden");
+        // Disable address-related radio buttons
+        Array.from(document.getElementsByClassName("depends-on-address")).forEach((element) => {
+            element.classList.add("cursor-not-allowed", "text-gray-500");
+            if (element instanceof HTMLInputElement) {
+                element.disabled = true;
+                element.checked = false;
+            }
+        });
+        // Show error
+        document.getElementById("no_address_found").classList.remove("hidden");
+    }
 }
 
-function construct_address(number: string, street: string, postcode: string, city: string, country: string) {
-    let address = ", " + postcode + ", " + city + ", " + country;
-
-    if (number.length != 0 && street.length != 0) {
-        address = street + " " + number + address;
-    } else if (street.length != 0) {
-        address = street + address;
-    } else if (number.length != 0) {
-        address = number + address;
-    } else {
-        address = address.substring(1);
+// Construct a single street string from street name and number
+function constructStreet(street: string, number: string) {
+    // Only append street number if street is given
+    if (street && number) {
+        return street + " " + number;
     }
-    return address;
+    return street;
 }
 
-function use_map_pin_adresse(
```

Comment:
To avoid too many conflicts with #1820, I'd suggest to use the proper naming conventions already even if they're not enforced yet. So in JS functions and variables, we're going to use CamelCase from now on...